README.md

```diff
@@ -9,16 +10,60 @@ This repository contains basic set of shared libraries and so-called SCR agents
 used for reading and writing configuration files and some even for executing
 commands on the system.
 
-## Installation ##
 
-    make -f Makefile.cvs
-    make
-    sudo make install
+Provided Functionality
+======================
 
-## Running Testsuites ##
+See [the generated yardoc documentation](http://www.rubydoc.info/github/yast/yast-yast2) at rubydoc.info.
 
-    make check
+Development
+===========
 
-## Links ##
+This module is developed as part of YaST. See the generic
+[development documentation](doc/README_Generic.md#yast-development-documentation) and
+[development environment](doc/README_Generic.md#development-environment) description.
 
-  * See more at http://en.opensuse.org/openSUSE:YaST_development
+
+Getting the Sources
+===================
+
+To get the source code, clone the GitHub repository:
+
+    $ git clone https://github.com/yast/yast-yast2.git
+
+Alternatively, you can fork the repository and clone your fork. This is most
+useful if you plan to contribute into the project and you do not have push
+permission to the repository.
```

Comment:
If we keep it this way, the first thing people would do, might be asking for permissions